app/actions/local/channel.ts

```diff
@@ -139,15 +139,17 @@ export async function setChannelDeleteAt(serverUrl: string, channelId: string, d
         const {operator, database} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
         const channel = await getChannelById(database, channelId);
         if (!channel) {
-            return;
+            return {error: `channel with id ${channelId} not found`};
         }
 
         const model = channel.prepareUpdate((c) => {
             c.deleteAt = deleteAt;
         });
         await operator.batchRecords([model], 'setChannelDeleteAt');
+        return {models: [model]};
     } catch (error) {
         logError('FAILED TO BATCH CHANGES FOR CHANNEL DELETE AT', error);
+        return {error};
```

Comment:
I updated this function to make it easier to test. I checked all usages of it and these changes shouldn't have any impact but let me know if I should revert it back.